core/src/scss/utilities/variables/core/z-index.scss

```diff
@@ -0,0 +1,34 @@
+@charset "UTF-8";
+
+//
+// Z-index
+//
+// Style guide: Variables.Core.Z-index
+//
+
+//
+// $level
+//
+// A sass map of z-indices
+//
+//```
+// $z-levels: (
+//   "navigation":  100,
+//   "button":      100,
+//   "masthead":     20,
+//   "overlay":      10,
+//   "default":       1,
+//   "background":  -100
+// );
+// ```
+//
+// Style guide: Variables.Core.Z-index.level
+//
+$z-levels: (
+  "navigation":  100,
+  "button":      100,
+  "masthead":     20,
+  "overlay":      10,
+  "default":       1,
+  "background":  -100
+);
```

Comment:
Are we adding the main nav z-indexes in here? Another good one to add might be the skiplinks (focus state) since that needs to be above everything in the masthead when it comes into focus. Looks like a great start 😄 👍 - thanks for doing this much needed wrok!